### PR TITLE
[FIX] web: fix popover in dialog with useAutofocus

### PR DESCRIPTION
**Pull request by brboi**
This PR also includes `[IMP] web: soften the UI active element takeover`

This is needed in order to make possible `[FIX] web: fix popover in dialog with useAutofocus`

### CODE REVIEW

File: addons/web/static/src/core/datetime/datetime_picker.xml [hunk 1]
```diff
@@ -27,6 +27,7 @@
                             <t t-set="arInfo" t-value="getActiveRangeInfo(itemInfo)" />
                             <button
                                 class="o_date_item_cell o_datetime_button o_center o_cell_md btn p-1 border-0 fw-normal"
+                                tabindex="-1"
```
Comment (fdardenne) [inline addons/web/static/src/core/datetime/datetime_picker.xml:30]: what's the purpose of the `tabindex` on buttons ?

**Review by fdardenne — COMMENTED** [review]
Can we make a test for the `useAutofocus` case on for popover in dialog https://github.com/odoo/odoo/pull/140885/commits/84649b1557d9f0fbf602b72d3764637a2c4987ac ?

File: addons/web/static/src/core/datetime/datetime_picker.xml (hunk 1 above)
Comment (FrancoisGe) [reply addons/web/static/src/core/datetime/datetime_picker.xml:30]: tabindex="-1" => not tabable

**Review by fdardenne — APPROVED** [review]
@robodoo r+

**Comment by fdardenne** [comment]
@robodoo rebase-ff

**Comment by fdardenne** [comment]
@robodoo r+

**Pull request closed without merging**